forge/routes/api/settings.js

```diff
@@ -26,6 +27,7 @@ module.exports = async function (app) {
             }
 
             if (request.session.User.admin) {
+                response['platform:licensed'] = isLicensed
```

Comment:
Not 100% sure about including this here as we already have the `/api/v1/admin/license` end point that returns license details.

That said, we can move forward with this as-is and iterate if needed.